OrderApi/OrderSettings.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace ShoesOnContainers.Services.OrderApi 
```

Comment:
I believe this class is not necessary if we don't use the IOptionsSnapshot settings in the controller, and we use the IConfiguration instead.